extensions/stackdriver/edges/edge_reporter.cc

```diff
@@ -68,15 +68,18 @@ void instanceFromMetadata(const ::wasm::common::NodeInfo& node_info,
 }  // namespace
 
 EdgeReporter::EdgeReporter(const ::wasm::common::NodeInfo& local_node_info,
-                           std::unique_ptr<MeshEdgesServiceClient> edges_client)
-    : EdgeReporter(local_node_info, std::move(edges_client), []() {
+                           std::unique_ptr<MeshEdgesServiceClient> edges_client,
+                           const int batch_size)
```

Comment:
nit: afaik `const int` is not really useful

Reply:
removed.